platform/app/services/create_connection_and_syncs_service.rb

```
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
```

rubocop

[convention] Incorrect indentation detected (column 0 instead of 2).

(Layout/CommentIndentation)